### ISSUE TITLE
Question about Multi-task Configuration

### ISSUE THREAD

**Issue by feralvam** [sweta20/ComplexityControlledMT#1]
Hi,
I just read your paper and it's a very nice work. I have a 'practical' question that I hope you can help me with. Where in the released code is the multi-task configuration? I couldn't find where you tell sockeye (using the command-line) to sum up the losses of each task before backprop. I'm not familiar with sockeye.

Thank you.

**Comment by sweta20** [comment]
Hi,

The way we do this is, we don't actually sum up the individual loss components as these are calculated on different datasets but share the same seq2seq model.

![Screenshot from 2019-11-09 11-00-18](https://user-images.githubusercontent.com/11375341/68521856-35659100-02e0-11ea-8ccd-1c588f35bfaa.png)

So, for example to train the multitask system illustrated in the diagram, you can use 
` bash scripts/main.sh -d -b -n -e -f -g -i <iter>`
You can check the description of the arguments using.
` bash scripts/main.sh -h`

I hope this makes it slightly clear.
